pointcloud/src/main/scala/geotrellis/pointcloud/raster/rasterize/triangles/PDALTrianglesRasterizer.scala

```diff
@@ -111,21 +109,13 @@ object PDALTrianglesRasterizer {
         val scancol0 = math.max(math.ceil((xmin - exmin) / w - 0.5), 0)
         var scanx = exmin + scancol0 * w + w / 2
         while (scanx < exmax && scanx < xmax) {
-          val col = ((scanx - exmin) / w).toInt
-          val row = ((eymax - scany) / h).toInt
-          if(0 <= col && col < cols &&
-            0 <= row && row < rows) {
-
+          val col = ((scanx - exmin) / w + w / 2).toInt
+          val row = ((eymax - scany) / h + h / 2).toInt
```

Comment:
@jpolchlo I am not sure this is an apporpriate fix, but it helped to remove seams we saw in the OGC server outputs.

Reply:
It didn't work however, I could create a test case